geowebcache/azureblob/src/test/java/org/geowebcache/azure/AbstractAzureBlobStoreIntegrationTest.java

```diff
@@ -154,7 +157,7 @@ public void testPutWithListener() throws MimeException, StorageException {
                         eq(tile.getLayerName()),
                         eq(tile.getGridSetId()),
                         eq(tile.getBlobFormat()),
-                        anyString(),
+                        anyStringOrNull(),
```

Comment:
These are changes in Mockito behavior, previous version was matching null too, the current one does not.